public/sw.js

```diff
@@ -19,7 +19,8 @@ self.addEventListener('install', event => {
 
 const CACHEABLE_FILES = [
   // Reverb
-  'https://mybbc-analytics.files.bbci.co.uk/reverb-client-js/reverb-3.9.2.js',
+  /^https:\/\/static(?:\.test)?\.files\.bbci\.co\.uk\/ws\/(?:simorgh-assets|simorgh1-preview-assets|simorgh2-preview-assets)\/public\/static\/js\/reverb\/reverb-.+\.js$/,
```

Comment:
Shall we hardcode the reverb version to 3.10.0?

Reply:
Sure thing.
I've set the version to [3.10.1](https://github.com/bbc/simorgh/pull/12879/files#diff-98fd1d4c44ad3934ee242b1f070d235cc39f11e46738f28de66c6740182c4706R22). This will come in handy whenever we
need to bump the version.